bundler/lib/bundler/man/bundle-version.1.ronn

```diff
@@ -0,0 +1,24 @@
+bundle-version(1) -- Prints the bundler's version information
+=============================================================
+
+## SYNOPSIS
+
+`bundle version`
+
+## DESCRIPTION
+
+Prints the bundler's version information.
```

Comment:
```suggestion
Prints Bundler version information.
```

I think the above reads slightly better, what do you think?

Reply:
Yes, makes sense to me. That sentence just came from:
https://github.com/rubygems/rubygems/blob/ff67cb0fba3526fa411c0fdf73269f22c46c7d9a/bundler/lib/bundler/cli.rb#L520
so updating the sentence in `cli.rb` as well.

Reply:
Oh, I see. Well, thanks for updating it!